dali/dfu/dfuutil.cpp

```diff
@@ -1265,11 +1266,17 @@ class CDFUhelper: implements IDFUhelper, public CInterface
     {
         DBGLOG("cloneRoxieSubFile src=%s@%s, dst=%s@%s, prefix=%s, ow=%d, docopy=false", srcLFN, srcCluster, dstLFN, dstCluster, prefix, overwriteFlags);
         CFileCloner cloner;
-        cloner.init(dstCluster, DFUcpdm_c_replicated_by_d, true, NULL, userdesc, foreigndali, NULL, NULL, false, false);
+        bool copyPhysical = false;
+        // MORE: Would the following be better to ensure files are copid when queries are deployed?
```

Comment:
Typo in comment

Reply:
I think we considered this for baremetal but the roxie copy mechanism was already working and seemingly preferred.  It's definitely worth considering for cloud.. although I'm not yet familiar enough with when we'd want to copying to storage planes to say for sure.  And I've also never tested the flag to see if it actually works.

Reply:
I tested the code when trying to debug the issue, and it did work.  It might be worth adding a flag to the service to indicate that physical files should be copied.

Reply:
I'm not familiar with how it normally works..
How does roxie see the physical files if they're copyPhysical=false , implying they are not copied at all?
Does that mean they are remotely accessed at the original source location?

Reply:
In baremetal depending on Roxies configuration it will copy the files itself in the background and access the files at the original source location until the copy is complete.  Roxie can also be configured to always access the files remotely, or to expect the files to have been copied through DFS.